cms/toolbar/items.py

```diff
@@ -286,7 +286,7 @@ class FrameItem(BaseItem):
     def __init__(self, name, url, active=False, disabled=False,
                  extra_classes=None, on_close=None, side=LEFT):
         super(FrameItem, self).__init__(side)
-        self.name = "%s ..." % force_text(name)
+        self.name = "%s..." % force_text(name)
```

Comment:
well, there should be no space between a word and the ellipsis in the ui, same as in all the applications or operating systems